src/todo/helper.lua

```diff
@@ -1,5 +1,10 @@
 
 function todo.log(message)
+    if type(message) == 'table' then
```

Comment:
That is a really nice addition :) Thanks

Reply:
I'm all about that quality of life for my fellow dev friends!